resources/frontend/inspect_network.md

```diff
@@ -0,0 +1,30 @@
+## Understanding Inspect Network
+The Network Panel allows you to inspect resources as they are accessed over the network. 
+
+### Opening Inspect Network
+    - Ctrl  + Shift + J to open the Console, and toggle "Network". Then, refresh your page. 
+
+### Inspect Network
+
+- **Filtering**
+    - If you are looking for a specific type of resource, you can filter the data using the tabs under
+    filter. (All, XHR, JS, CSS, Img, etc) Most of these of self-explanatory except for XHR. 
+    #### XHR
+    -  In this tab, you will be able to check the requested API endpoints. You can select the Name of the resource 
+    to see what is included in the API endpoint. This allows you to check if the API is getting the correct information. It makes returning an object and its atrribrutes a lot         easier because all the attribrutes and their values are listed out for each object. 
+    For example: 
```

Comment:
Can we get a few screenshots too here as well? You can place the images into resources/images